### ISSUE TITLE
grade to see this project, and how is the accurancy achieved?

### ISSUE THREAD

**Issue by flankechen** [sophiaTeng/LRF#1]
I just read the paper and thinking about coding this.
I am trying to run your code and wondering how is the accurancy?

**Comment by sophiaTeng** [comment]
a little higher than the paper

**Comment by flankechen** [comment]
Thanks for your reply, I run your code and in the testing part, traverseComLRT.m, in the division node. why you are adding only the x and y for the 3d offset theta? and resulting all the test position z unchanged?

and looks like you just check the x y distance?

Thanks again.

**Comment by sophiaTeng** [comment]
yes, I ignore the the z position, and only compare the x y position
